.circleci/config.yml

```diff
@@ -43,18 +43,23 @@ jobs:
     steps:
       - attach_workspace:
           at: ./
+      - run:
+          name: Set registry URL
+          command: npm set registry https://npm.jahia.com/
       - run:
           name: Authenticate with NPM registry
-          command: echo '//npm.jahia.com/:_authToken=$NPM_TOKEN' >>  `npm config get userconfig`
+          command: echo '//npm.jahia.com/:_authToken=$NPM_TOKEN' >> .npmrc
```

Comment:
Can you also add a `cat .npmrc` command to check the value ? the token may appear in the log , we may have to delete it and create a new one after ..